modules/core/src/main/java/org/apache/ignite/internal/GridTopic.java

```diff
@@ -81,6 +81,12 @@
     /** */
     TOPIC_CONTINUOUS,
 
+    /**
+     * @deprecated Should be removed in Apache Ignite 3.0.
```

Comment:
"Not used anymore. Must not be removed until Apache Ignite 3.0 release to preserve binary protocol compatibility."